Assets/Treasure/Example/Scripts/IdentityUI.cs

```diff
@@ -45,4 +47,73 @@ public async void OnAuthTokenSubmit(string value)
             TDKLogger.LogError($"Error validating user session: {e.Message}");
         }
     }
+
+    public async void OnGetMagicBalanceBtn()
+    {
+        var contractAddress = await TDK.Common.GetContractAddress(Treasure.Contract.Magic);
+        var contract = TDKServiceLocator.GetService<TDKThirdwebService>().SDK.GetContract(contractAddress);
+        var balance = await contract.Read<string>("balanceOf", TDK.Identity.Address);
+        TDKLogger.LogInfo($"Magic balance: {Utils.ToEth(balance)}");
+    }
+
+    public async void OnMintMagicBtn()
+    {
+        TDKLogger.LogInfo("Minting 1,000 MAGIC...");
+        var transaction = await TDK.API.WriteTransaction(Treasure.Contract.Magic, "mint", new object[] { TDK.Identity.Address, Utils.ToWei("1000") });
+        transaction = await TDK.Common.WaitForTransaction(transaction.queueId);
+        if (transaction.status == "errored")
+        {
+            TDKLogger.LogError($"Mint failed: {transaction.errorMessage}");
+        }
+        else
+        {
+            TDKLogger.LogInfo($"Mint successful: {transaction.transactionHash}");
+        }
+    }
+
+    public async void OnRawMintMagicBtn()
```

Comment:
Is there a difference between doing it "raw" here vs the above? or is it just as an usage example of the new `SendRawTransaction` function?

Reply:
Just a usage example